doc/camera_and_viewport.md

```diff
@@ -0,0 +1,45 @@
+# Camera and Viewport
+
+When rendering on Flutter, the regular coordinate space used are logical pixels. That means one pixel for Flutter is already not necessarily one real pixel on the device, because of the [device's pixel ratio](https://api.flutter.dev/flutter/widgets/MediaQueryData/devicePixelRatio.html). When it gets to the Flame level, we always consider the most fundamental level to be logical pixels, so all the device specific complexity is abstracted away.
```

Comment:
Max line length of 100 to be in line with the other docs

Reply:
really? didn't know about this rule, is it enforced by our linters or written anywhere?

Reply:
It's not, we don't have an md linter, me and @wolfenrain talked about introducing one though

Reply:
Yeah I have been working on enforcing it globally, hope to have something for it soon :-)

Reply:
is there an easy way to format for now or do I have to manually split the lines?

Reply:
figured it out!